src/main/java/com/coveo/pushapiclient/Sandbox.java

```diff
@@ -0,0 +1,88 @@
+package com.coveo.pushapiclient;
+
+import java.util.HashMap;
+import java.util.List;
+
+public class Sandbox {
```

Comment:
Would this class be internal/not exported to sdk consumers?

Reply:
No. this is just a sample file. For the sake of the examples.
I did not include it in the `samples/` folder so anyone could checkout the branch and play around in that file and still get auto-completions